crosscluster/logical: don't set fraction once resolved ts exists

dt opened the pull request:
Release note: none.
Epic: none.

dt committed:
crosscluster/logical: don't set fraction once resolved ts exists

Release note: none.
Epic: none.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/134721)
<!-- Reviewable:end -->

dt committed:
crosscluster/producer: switch timer to ticker to continue polling

Also while we are here re-read walltime per range in case a large number of
ranges takes a long time to iterate to ensure ranges near the end of iteration
do not see artificial lag.

Release note: none.
Epic: none.

dt committed:
ui/jobs: show running status for LDR jobs

Release note: none.
Epic: none.

dt committed:
crosscluster/logical: elide status msg when caught up

Release note: none.
Epic: none.

dt commented:
TFTR!

bors r+

dt commented:
flake: 
```
    test_server_shim.go:152: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:90: CREATE TABLE public.table10 (col10_0 INT8 NOT NULL, col10_1 BPCHAR NOT NULL, col10_2 STRING COLLATE da_DK NOT NULL, col10_3 PG_LSN NOT NULL, col10_4 STRING NOT NULL AS (lower(CAST(col10_2 AS STRING))) VIRTUAL, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table10_pkey PRIMARY KEY (rowid ASC), INDEX table10_col10_4_col10_1_col10_3_col10_0_idx (col10_4 DESC, col10_1 ASC, col10_3 DESC, col10_0 DESC) PARTITION BY LIST (col10_4, col10_1) (PARTITION table10_part_0 VALUES IN ((e'\x00', '<'), (e'>`b\x12)e\b4', 'Q'), ('9PG', 'u'), ('i', ','), ('p', 'Z'))), INDEX table10_col10_1_col10_4_idx (col10_1 ASC, col10_4 ASC), CONSTRAINT table10_col10_4_key UNIQUE (col10_4 ASC), INVERTED INDEX table10_col10_2_col10_4_col10_1_idx (col10_2 ASC, col10_4 DESC, col10_1 gin_trgm_ops), INDEX table10_col10_1_expr_idx (col10_1 DESC, lower(CAST(col10_2 AS STRING)) ASC) STORING (col10_2, col10_3) PARTITION BY LIST (col10_1) (PARTITION table10_part_0 VALUES IN (('T'), ('K'), ('!'), (':'), (e'\x1a'), (e'\x19'), ('['), ('z')), PARTITION table10_part_1 VALUES IN (('/'), ('"'), ('m'), ('Z'), ('2'), (','), ('H')), PARTITION table10_part_2 VALUES IN ((e'\x1f'), ('|'), ('w'), ('1'), ('P'), ('l'), (e'\U00002603')), PARTITION table10_part_3 VALUES IN ((')'), ('.'), (e'\x13'), (e'\x18'), (''), ('y')), PARTITION table10_part_4 VALUES IN ((e'\x0b'), ('4'), ('v'), ('+'), ('-'), ('O')), PARTITION table10_part_5 VALUES IN (('0'), ('k'), ('V'), ('X'), ('%'), ('R')), PARTITION table10_part_6 VALUES IN ((e'\''), ('W'), ('#'), ('N'), ('p'), (' ')), PARTITION table10_part_7 VALUES IN (('Q'), (e'\x12'), (e'\x1e')), PARTITION "DEFAULT" VALUES IN ((DEFAULT))), INDEX table10_col10_4_idx (col10_4 ASC) STORING (col10_0, col10_1, col10_2, col10_3), INDEX table10_col10_1_idx (col10_1 ASC) STORING (col10_0, col10_2, col10_3), CONSTRAINT table10_col10_1_key UNIQUE (col10_1 DESC) STORING (col10_0), CONSTRAINT table10_col10_4_key1 UNIQUE (col10_4 ASC) STORING (col10_0, col10_2), FAMILY fam_0_col10_1_col10_3_col10_0_rowid (col10_1, col10_3, col10_0, rowid), FAMILY fam_1_col10_2 (col10_2)) pq: relation "table10" (129): ('') cannot be present in more than one partition
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema2563455851
--- FAIL: TestCreateRandomSchema (1.53s)
```

bors r+
